### ISSUE TITLE
Adding a repository for a package with an unknown repository does not remove it from the "Failed to analyze" list

### ISSUE THREAD

**Issue by fniessink** [deadpendency/deadpendency#34]
I have a few packages that deadpendency fails to find the GitHub repo for, e.g. bandit. I would expect that adding:

```yaml
additional-deps:
  python:
    - name: bandit
      repo: PyCQA/bandit
```
would remove bandit from the list of "Failed to analyze" packages, but it doesn't.

**Comment by AlistairB** [comment]
Thanks for the bug report!

Yes that should work oops. I should have a fix out in a day or two.

**Comment by AlistairB** [comment]
Thanks @fniessink . This should be fixed as part of `0.3.15.0` update.

With the bug resolved it produced this report for your repo https://github.com/deadpendency/quality-time/runs/2415991881

Feel free to reopen if you think there are still issues.

**Comment by fniessink** [comment]
Thanks for the prompt response and fix!
